ocs_ci/ocs/ui/base_ui.py

```diff
@@ -248,6 +248,16 @@ def get_elements(self, locator):
         """
         return self.driver.find_elements(by=locator[1], value=locator[0])
 
+    def get_element_attribute(self, locator, attribute, safe: bool = False):
+        """
+        Get attribute from WebElement
+        """
+        el = self.get_elements(locator)
```

Comment:
el can be replaced with a better variable

Reply:
done, renamed to web_elements